### PR TITLE
docs(tutorial): PR fixes for issue credential

### PR DESCRIPTION
**Pull request by blu3beri**
- Fixes most of the feedback that @TimoGlastra left on the previous PR
- I will create a new PR after this is merged with the updated code snippets

**IMPORTANT**

The code snippets are incorrect and with the next PR these will be extracted and fixed.

### CODE REVIEW

File: guides/tutorials/issue-a-credential.md [hunk 1]
```diff
@@ -37,6 +37,13 @@ a [Node.js environment](../getting-started/prerequisites/nodejs).
 For the _Holder_ we need to setup a basic agent with a wallet, mediator,
 outbound transport and a ledger.
 
+It is very important to note that mobile agents do not support HTTP by default.
+It is recommended to do everything over HTTPS, but for development HTTP can be
+enabled for
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:42]: how do we determine line wrapping? This line only contains two words

File: guides/tutorials/issue-a-credential.md [hunk 2]
```diff
@@ -211,17 +259,12 @@ import { CredentialProtocolVersion, CredentialPreviewAttribute } form '@aries-fr
 const credentialExchangeRecord = agent.credentials.offerCredential({
   connectionId,
   protocolVersion: CredentialProtocolVersion.V1,
+  credentialDefinitionId: credentialDefinition.id,
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:262]: credentialDefinitionId is part of the `indy` key

File: guides/tutorials/issue-a-credential.md [hunk 3]
```diff
@@ -240,17 +283,12 @@ import { CredentialProtocolVersion, CredentialPreviewAttribute } form '@aries-fr
 const credentialExchangeRecord = agent.credentials.offerCredential({
   connectionId,
   protocolVersion: CredentialProtocolVersion.V2,
+  credentialDefinitionId: credentialDefinition.id,
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:286]: same here

File: guides/tutorials/issue-a-credential.md [hunk 4]
```diff
@@ -436,17 +475,12 @@ const run = async () => {
   const credentialExchangeRecord = agent.credentials.offerCredential({
     connectionId,
     protocolVersion: CredentialProtocolVersion.V1,
+    credentialDefinitionId: credentialDefinition.id,
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:478]: same here

File: guides/tutorials/issue-a-credential.md [hunk 5]
```diff
@@ -545,17 +580,12 @@ const run = async () => {
   const credentialExchangeRecord = agent.credentials.offerCredential({
     connectionId,
     protocolVersion: CredentialProtocolVersion.V2,
+    credentialDefinitionId: credentialDefinition.Id,
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:583]: same here

File: guides/tutorials/issue-a-credential.md [hunk 6]
```diff
@@ -545,17 +580,12 @@ const run = async () => {
...
         attributes: [
-          new CredentialPreviewAttribute({
-            name: "Name",
-            value: "Jane Holder",
-          }),
-          new CredentialPreviewAttribute({
-            name: "Date of birth",
-            value: "10-08-1990",
-          }),
+          { name: "Name", value: "Jane Holder" },
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:587]: this is much better 👍

File: guides/tutorials/issue-a-credential.md (hunk 2 above)
Comment (blu3beri) [reply guides/tutorials/issue-a-credential.md:262]: Snippets will be fixed in the next pr.

**Comment by blu3beri** [comment]
@TimoGlastra could you approve it if the only issue was the formatting? I have removed the snippets as they will be fixed in #21

**Pull request merged**